tests/e2e-test/tests/e2e_test.go

```diff
@@ -13,18 +13,22 @@ import (
 //
 // This driver expects a directory '.expected' in the top level of the package and
 // '.expected' should contain 3 files:
-//  - 'exitcode.txt' contains a single number which is expected exit code of command
-//    'kpt fn run'. If this file is missed, driver will assume the expected exit code
-//    is 0.
+//  - 'config.yaml' has the configuration information for this test. It should contain a
+//    map of following fields:
+//    - 'exitCode': Contains a single number which is expected exit code of command
+//      'kpt fn run'. If this field is missed, driver will assume the expected exit code
+//      is 0.
+//    - 'network': A boolean which indicates whether network should be enabled
+//    	for this test. If this field exists and the content in it is 'true' then the
+//    	network is accessible. Otherwise the function cannot access network.
+//    - 'runTimes': A number which sepcifies the times of function running. If this field
+//      missing then the function will be run once.
```

Comment:
Should we run more than once by default?

Reply:
Updated the default value to 2.